ocamldoc/odoc_to_text.ml

```diff
@@ -144,6 +144,18 @@ class virtual info =
         []
         l
 
+    method text_of_alerts alerts =
+      List.concat_map (fun al ->
+          let payload =
+            match al.alert_payload with
+            | Some p -> [ Raw ". "; Raw p ]
+            | None -> []
+          in
+          [ Bold [ Raw Odoc_messages.alert; Raw al.alert_name ] ]
```

Comment:
You need a white space between `Alert` and the alert name here.

Reply:
Fixed.